.devcontainer/devcontainer.json

```diff
@@ -47,6 +51,35 @@
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/esm/msg/",
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/ies/",
 			"-I/workspaces/magma/lte/gateway/c/oai/tasks/nas/util/"
+		],
+		"files.watcherExclude": {
+			"**/.bazel-cache/**": true,
+			"**/.bazel-cache-repo/**": true,
+		},		
+		"bsv.bazel.buildFlags": [
+			"--config=devcontainer",
+		],
+		"bsv.bazel.testFlags": [
+			"--compilation_mode=dbg",
+		],
+		"bsv.bes.enabled": false,
+		"bsv.bzl.codesearch.enabled": false,
+		"bsv.bzl.invocation.buildEventPublishAllActions": false,
+		"bsv.bzl.invocation.enabled": false,
+		"bsv.bzl.invocation.invokeWithBuildEventStreaming": false,
```

Comment:
Do you generally have a pointer for how you determined all these configuration parameters? Or did the author pass them to you?

Anything here that may change down the road? Would be nice to have comments in PR description.